src/patients/GeneralInformation.tsx

```diff
@@ -273,27 +230,28 @@ const NewPatientForm = (props: Props) => {
               name="address"
               value={patient.address}
               isEditable={isEditable}
-              onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) => {
-                onFieldChange('address', event.currentTarget.value)
-              }}
+              onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) =>
+                onFieldChange && onFieldChange('address', event.currentTarget.value)}
             />
           </div>
         </div>
-        {isEditable && (
-          <div className="row float-right">
-            <div className="btn-group btn-group-lg">
-              <Button className="mr-2" color="success" onClick={onSaveButtonClick}>
-                {t('actions.save')}
-              </Button>
-              <Button color="danger" onClick={() => onCancel()}>
-                {t('actions.cancel')}
-              </Button>
-            </div>
-          </div>
-        )}
-      </form>
+      </Panel>
+      {isEditable ? (
```

Comment:
Github thinks this file is NewPatientForm.tsx (which didn't have panels) renamed to GeneralInformation.tsx, when in fact I deleted NewPatientForm.tsx, so that's why it thinks I put panels in.